### PR TITLE
Fix PED Bugs

### LINKED ISSUE #139: [PE-D] Hyperlinks in User Guide
Great job in making sure that the user guide is navigable!

However, there are some hyperlinks that are not directing to the correct sections (specifically in the "Shortcut to Specific Features")

Some of the features with broken hyperlinks include:
- Add Status to a Contact
- Unarchive
- Switch

-------------
Labels: `severity.Low` `type.DocumentationBug`
original: Korochii/ped#1

### LINKED ISSUE #166: [PE-D] Commands that uses INDEX
Not really a bug but I feel it will be good if the user guide (eg. clearmodules) specifies that the index must be within the range for it to be valid (on top of the existing note about how "The index must be a positive integer 1, 2, 3, …​".

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: Korochii/ped#9

### LINKED ISSUE #167: [PE-D] Inconsistency in stated save location
**What the bug is about/steps**<br>
Indicated in the UG that the file is saved to `ModuleMateFinder.json` However, the application window and actual saved files state otherwise

**UG content**<br>
 <br>

**File name saved**<br>
 <br>

**File path stated in window**<br>

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: charmainehly/ped#8

### LINKED ISSUE #183: [PE-D] Missing Commands in Command Summary
Great job in ensuring that the command summary is clear, examples are included as well which can be quite useful for users.
However, I noticed that there are some commands that are missing in the summary, such as `exit`, `archive` and `help`.

-------------
Labels: `type.DocumentationBug` `severity.Medium`
original: Korochii/ped#13

### LINKED ISSUE #187: [PE-D] Format for module code not specified
No format was provided in the user guide for module code

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: redpelican2108/ped#7

### PR DESCRIPTION
**Pull request by fantablack**
### The following bugs are addressed:
* Specify Module code format (closes #187)
* Fix broken link to annotated image of example person
* Update and reorder command summary list (closes #183)
* Rename Command headers to be clearer of their purpose
* Fix broken links in Command Summary (closes #139)
* Update savedata locations in UG (closes #167)
* Update valid Index description for all Commands (closes #166)

### CODE REVIEW

**Comment by bakano98** [comment]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -144,14 +144,21 @@ Adds module(s) to an existing contact
 Format: `addmodule INDEX m/MODULE [m/MODULE]...`
 
 * Adds modules represented by each module code `m/MODULE` to a contact at index `INDEX`
-* The index refers to the index number shown in the displayed contact list.
-* The index **must be a positive integer** 1, 2, 3, …​
+* The `Index` refers to the index number shown in the displayed contact list.
```
Comment (bakano98) [inline docs/UserGuide.md:147]: I think should capitalise `Index`, more consistent with what we have above since we are formatting it.

**Review by bakano98 — COMMENTED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (fantablack) [reply docs/UserGuide.md]: Good point. Updated!

**Comment by junlee1991** [comment]
LGTM

**Pull request merged**